home/import_content_pages.py

```diff
@@ -8,7 +8,10 @@
 from typing import Any
 from uuid import uuid4
 
-from django.core.exceptions import ObjectDoesNotExist, ValidationError  # type: ignore
+from django.core.exceptions import (
+    ObjectDoesNotExist,  # type: ignore
+    ValidationError,
```

Comment:
I don't think this should've changed, it's causing a typing error in the linting now